resources/quality/mixware_hyper_k/mixware_hyper_k_draft.inst.cfg

```
@@ -0,0 +1,17 @@
+[general]
+version = 4
+name = Draft
+definition = mixware_hyper_k
+
+[metadata]
+type = quality
+quality_type = draft
+global_quality = True
+setting_version = 20
+
+[values]
+layer_height = 0.15
+layer_height_0 = 0.15
+top_bottom_thickness = =layer_height_0+layer_height*6
```

```suggestion
top_bottom_thickness = =layer_height_0 + layer_height * 6
```
Please use spaces around operators. I've not marked them everywhere, as that would be quite a bit of work.